CHANGELOG.md

```diff
@@ -89,6 +89,7 @@
 - `mongodbreceiver`: Establish codebase for MongoDB metrics receiver (#6972)
 - `couchbasereceiver`: Establish codebase for Couchbase metrics receiver (#7046)
 - `dbstorage`: New experimental dbstorage extension (#7061)
+- `k8seventsreceiver` (#6885)
```

Comment:
Please move it to `Unreleased` section and rename to 
```
- `k8seventsreceiver`: Implement core logic (#6885)
```

Reply:
I have updated the file.